__test__/integration/debt_token_api/runners/index.ts

```diff
@@ -75,6 +75,19 @@ export abstract class ScenarioRunner {
         return new BigNumber(tokenIDAsString);
     };
 
+    public generateDebtTokenIDWithoutFulfillment = async (
```

Comment:
I think this naming could be clearer -- maybe `getDebtTokenIDFromUnfilledOrder`?